### PR TITLE
Update Readme file to add a notice about API proposal

### PR DESCRIPTION
**Pull request by bigludo7**
Proposal for https://github.com/camaraproject/CarrierBillingCheckOut/issues/46

Please do not hesitate to add/correct directly in the text.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,11 +8,25 @@
 # CarrierBillingCheckOut
 Repository to describe, develop, document and test the CarrierBillingCheckOut API family
 
+## Notice for the reader
+The Carrier Billing familly features 2 APIs with **distinct business scope**:
+* 'Carrier billing checkout' : Service that provides functionality for enabling the first the purchase and then payment request, and follow up of fulfilment of goods against OB carrier billing systems.
```
Comment (PedroDiez) [inline README.md:13]:
```suggestion
* 'Carrier billing checkout' : Service that provides functionality for enabling the first the purchase and then payment request, and follow up of fulfilment of goods against OB Carrier Billing Systems.
```

**Review by PedroDiez — APPROVED** [review]
Thanks for the contribution
Directly commited some editorials Ludovic

LGTM

**Comment by bigludo7** [comment]
Thanks Pedro for the review. I let one day to other team member and will merge tomorrow ;)

File: README.md [hunk 2]
```diff
@@ -8,11 +8,25 @@
...
+
 ## Scope
 * Service APIs for “CarrierBillingCheckOut” (see APIBacklog.md)  
 * It provides the customer with the ability to:  
-  * purchase, pay and follow up of fulfilment of physical/digital goods, using as Payment Method Carrier Billing, i.e.: the operator performs the billing of the goods.
-  * NOTE: The scope of this API family should be limited (at least at a first stage) to 4G and 5G.
+  * purchase (only for 'carrier billing checkout' flavor),
+  * trigger carrier billing payment request (in one or two steps) 
+  * follow up of payment processing using as Payment Method Carrier Billing, i.e.: the operator performs the billing of the goods.
+  * follow up of fulfilment of physical/digital goods (only for 'carrier billing checkout' flavor),
```
Comment (PedroDiez) [inline README.md:28]:
```suggestion
```

Comment (PedroDiez) [reply README.md:28]: let's removed this. 

Internally checked and aligned with Juan Carlos García comment

**Comment by PedroDiez** [comment]
@bigludo7 Ludivic let's wait until Monday EoD, if no additional comments let's merge it

**Pull request merged**